kong/plugins/rate-limiting/handler.lua

```diff
@@ -188,9 +188,15 @@ function RateLimitingHandler:access(conf)
 
     -- If limit is exceeded, terminate the request
     if stop then
-      return kong.response.error(429, "API rate limit exceeded", {
-        [RETRY_AFTER] = reset
-      })
+       if conf.response_body then
+          return kong.response.error(429, conf.response_body, {
+            [RETRY_AFTER] = reset
+          })
+       else
+          return kong.response.error(429, "API rate limit exceeded", {
+            [RETRY_AFTER] = reset
+          })
+       end
```

Comment:
Try to avoid repetitions, as they become error prone and add to maintenance burden.

```suggestion
       return kong.response.error(429, conf.response_body or "API rate limit exceeded", {
         [RETRY_AFTER] = reset
       })
```

Reply:
This looks beautiful "javierguerragiraldez", Thanks